New version: GoFish v0.3.0

JuliaRegistrator opened the pull request:
- Registering package: GoFish
- Repository: https://github.com/itsdfish/GoFish.jl
- Created by: @itsdfish
- Version: v0.3.0
- Commit: c829bb493c7f95dbfb694681837c17055264cc13
- Reviewed by: @itsdfish
- Reference: https://github.com/itsdfish/GoFish.jl/issues/4#issuecomment-2724070086
- Description: A Julia package for playing and simulating the card game Go Fish
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes

- Changed PlayGame to InteractiveGame
- call deal! inside simulate!
...
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598db58ab862640e20b47c6dbc1f9f1340509f4f851689fd77f4fda82ce3390817f7f1e45f3e22d994ffe32600c7e1d72a5f2ea64b2ac1540be06c984507e115a2fd3a1cd515e7394165ce2f458e0ef9d6d7d6dfc31b1c2e2ccb779420ef5ef934631e58c085be252e5462dd9074b6b80117eab384fb45ceaadbc70722cfb889361 -->

itsdfish commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

- Changed PlayGame to InteractiveGame
- call deal! inside simulate!
...

JuliaRegistrator committed:
New version: GoFish v0.3.0

UUID: d00ce6d8-e47b-41eb-9812-442785832f3b
Repo: https://github.com/itsdfish/GoFish.jl.git
Tree: 4c68fbd4fbffa9022a07d96e949fe103d3f2bc0f

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
